pkg/utils/utils.go

```diff
@@ -9,20 +9,18 @@ import (
 	"time"
 )
 
-var urand *rand.Rand
-
-func init() {
-	urand = rand.New(rand.NewSource(time.Now().UTC().UnixNano()))
```

Comment:
What about using the global random source and just calling `rand.Seed(time.Now().UTC().UnixNano())` in `init`?